packages/utils/src/utils/arrays.utils.ts

```diff
@@ -10,6 +10,31 @@ export const uint8ArrayToBigInt = (array: Uint8Array): bigint => {
   }
 };
 
+export const bigIntToUint8Array = (value: bigint): Uint8Array => {
```

Comment:
that function is refactored from somewhere right? i.e. code is move and somewhere the code is replaced by a call to this function

feels like we should have it somewhere but don't see that in the PR, maybe I missed it

Reply:
No, it's new here. Or at least I didn't find it anywhere and created it. Actually, it's funny because Github Copilot wrote it after I wrote the name of the function.

Reply:
Thanks, you are right, it's in [agent-js](https://github.com/dfinity/agent-js/blob/3a907af08bb8c338c00f48c3032f15c82ed297bf/packages/agent/src/agent/http/types.ts#L112) for something else. Somewhere I remembered that Maks patched it. All good then.